validator.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+func main() {
+	var conf, doc string
+	flag.StringVar(&conf, "conf", "file:///schema.json", "Schema file")
+	flag.StringVar(&doc, "doc", "file:///examples/invalid/invalid_email.json", "Example file")
```

Comment:
The actual flags never get parsed:
`flag.Parse()` to actually make that configurable.
This will also enable "go run validator.go --help"